README.md

```diff
@@ -76,3 +76,38 @@ $ docker run --tty \
     --env S3_FORCE_OVERWRITE="1" \
     --publish="80:80" tile-server tiles
 ```
+
+### Publishing Tiles through AWS Batch
+
+To generate tiles for production usage we use [AWS Batch tool](https://aws.amazon.com/batch/), which allows to execute multiple similar jobs in parallel. In order to use AWS Batch, the first prerequisite is to push the docker image into AWS ECR Repository. Tile server image is already there so this step is only required in case if you have done any changes to the image (that is, changes to dockerfiles itself or to the files used by it). In order to push the image to ECR Repository, execute following commands: 
+
+* `$(aws ecr get-login --no-include-email --region us-east-1)`: login to ECR
+* `docker build -t tile-server .`: build tile server docker image
+* `docker tag tile-server:latest 434035161053.dkr.ecr.us-east-1.amazonaws.com/tile-server:latest`: tag the image with the version called "latest"
+* `docker push 434035161053.dkr.ecr.us-east-1.amazonaws.com/tile-server:latest`: push (upload) docker image to the repository
+
+These steps should normally be set up to run in CI system, but at the time of writing this we don't have one, so they need to be executed manually. 
```

Comment:
do we have a ticket to do this?

Reply:
@ianwestcott has it: https://app.asana.com/0/1113179098808463/1116455779572731/f